src/lib.rs

```diff
@@ -91,6 +91,9 @@ use log::{debug, info, trace, warn};
 use rusqlite::NO_PARAMS;
 use rusqlite::{Connection, OptionalExtension, Transaction};
 
+/// Support for asynchonous migrations using tokio-rusqlite.
+#[cfg(feature = "async-tokio-rusqlite")]
+pub mod asynch;
```

Comment:
Perhaps it would be more ergonomic not to expose this module publicly, just the `AsyncMigrations` struct, what do you think?

Reply:
I have no strong feelings :).

Reply:
Hum, then I prefer having the module private 😃